### DISCUSSION TITLE
Ignore a specific mutation on a specific line

### DISCUSSION THREAD

**Discussion by ems-batkin** [stryker-mutator/stryker-net#3145] [Q&A] [answered]
Sometimes I run into mutations that are just alternative but correct code.

For example, if I have a method that takes a list of integers and finds the median value, I would sort the list ascending and then retrieve the element in the middle or average the two middle elements if the list has an even number. Stryker will mutate the function to sort the list _descending_ and then give me a smug look while chastising me for not killing it, but sorting descending is still valid. The elements in the middle are still the elements in the middle either way. Is there a way of ignoring that specific mutation on that line of code, or do I need to ignore the entire method?

**Comment by rouke-broersma** [comment, marked as answer]
https://stryker-mutator.io/docs/stryker-net/ignore-mutations/#stryker-comments

**Reply by daleardi** [reply]
thank you!
